tools/kompat/README.md

```diff
@@ -5,11 +5,11 @@ Kompat is a simple CLI tool to interact with `compatibility.yaml` files which ho
 ## Installation 
 
 ```
-go install github.com/aws/karpenter/tools/kompat
+go install github.com/aws/karpenter/tree/main/tools/kompat
```

Comment:
```suggestion
go install github.com/aws/karpenter/tools/kompat@latest
```

The original path is correct, but adding @latest at the end would probably help this from failing